packages/houdini-svelte/src/runtime/stores/pagination/offset.ts

```diff
@@ -61,14 +61,27 @@ export function offsetHandlers<_Data extends GraphQLObject, _Input extends {}>({
 				throw missingPageSizeError('loadNextPage')
 			}
 
+			// Get the Pagination Mode
+			let isPageByPage = artifact.refetch?.mode === 'PageByPage'
+
 			// send the query
-			await fetchUpdate({
-				variables: queryVariables as _Input,
-				fetch,
-				metadata,
-				policy: CachePolicy.NetworkOnly,
-				session: await getSession(),
-			})
+			if (isPageByPage) {
+				await parentFetch({
+					variables: queryVariables as _Input,
+					fetch,
+					metadata,
+					policy: CachePolicy.NetworkOnly, // we want to use the cache policy of the artifact, not this! :o
+					// session: await getSession(), // Hum?
+				})
+			} else {
+				await parentFetchUpdate({
+					variables: queryVariables as _Input,
+					fetch,
+					metadata,
+					policy: CachePolicy.NetworkOnly,
+					session: await getSession(),
+				})
+			}
```

Comment:
we still want the session passed here. maybe something like:

```suggestion
			const targetFetch = isPageByPage ? parentFetch :  parentFetchUpdate
			await targetFetch({
				variables: queryVariables as _Input,
				fetch,
				metadata,
				policy: isPageByPage ?  artifact.policy : CachePolicy.NetworkOnly,
				session: await getSession(), // Hum?
			})
```